tools/cpp/CMakeLists.txt

```diff
@@ -28,30 +28,10 @@ project (generate_geocoding_data)
 include (gtest.cmake)
 include (FetchContent)
 
-if(NOT absl_FOUND)
-  # Downloading the abseil sources at particular version to not catch up
-  # with its new build requirements like min C++14 is mandated in that lib.
-  FetchContent_Declare(
-      abseil-cpp
-      GIT_REPOSITORY  https://github.com/abseil/abseil-cpp.git
-      GIT_TAG         273292d
-  )
-
-  # Building the abseil binaries
-  FetchContent_GetProperties(abseil-cpp)
-  if (NOT abseil-cpp_POPULATED)
-      FetchContent_Populate(abseil-cpp)
-  endif ()
-
-  if (NOT abseil-cpp_POPULATED)
-     message (FATAL_ERROR "Could not build abseil-cpp binaries.")
-  endif ()
-
-  # Safeguarding against any potential link errors as mentioned in
-  # https://github.com/abseil/abseil-cpp/issues/225
-  set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
-  add_subdirectory(${abseil-cpp_SOURCE_DIR} ${abseil-cpp_BINARY_DIR})
-endif()
+# Safeguarding against any potential link errors as mentioned in
+# https://github.com/abseil/abseil-cpp/issues/225
```

Comment:
For product that uses abseil present/installed in the system, having the reference as a git module may not be pointing to right version..  Eventhough there is a way to configure version, it may not be the exact one used i.e no validations happening. How do we solve such cases? I do not have much experience in this area.

Reply:
We could also put it into a `if(NOT absl_FOUND)` such that the submodule is only used when it is not present in the system.